Ginger/Ginger/Variables/DatabaseListViewHelper.cs

```diff
@@ -0,0 +1,386 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.UserControls;
+using Ginger.Environments;
+using Ginger.UserControlsLib.UCListView;
+using GingerCore.DataSource;
+using GingerCore.Environments;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace Ginger.BusinessFlowPages.ListHelpers
+{
+    public class DatabaseListViewHelper : IListViewHelper
+    {
+        private readonly ObservableList<IDatabase> Databases = new ObservableList<IDatabase>();
+        private readonly Context mContext;
+
+        public delegate void DatabaseListItemEventHandler(ListItemEventArgs EventArgs);
+        public event DatabaseListItemEventHandler DatabaseListItemEvent;
+        private void OnActionListItemEvent(ListItemEventArgs.eEventType eventType, Object eventObject = null)
+        {
+            DatabaseListItemEventHandler handler = DatabaseListItemEvent;
+            if (handler != null)
+            {
+                handler(new ListItemEventArgs(eventType, eventObject));
+            }
+        }
+
+        public DatabaseListViewHelper(ObservableList<IDatabase> Databases, Context context) 
+        {
+            this.Databases = Databases;
+            this.mContext = context;
+        }
+        private UcListView mListView = null;
+        public UcListView ListView {
+            get
+            {
+                return mListView;
+            }
+            set
+            {
+                mListView = value;
+            }
+        }
+
+        public General.eRIPageViewMode PageViewMode { get; set ; }
+        public bool AllowExpandItems { get; set; } = true;
+        public bool ExpandItemOnLoad { get; set; } = true;
+
+        public bool ShowIndex { get { return false; } }
+
+        public void CopySelected()
+        {
+            throw new NotImplementedException();
+        }
+
+        public void CutSelected()
```

Comment:
Make sure menu is not being displayed for the operations which are throwing Not Implemented exception

Reply:
Yes, I have not added the operations in the list items for the dropdown itself